src/locales/en.json

```diff
@@ -316,41 +307,41 @@
       "instruction": "Only your face can be in the selfie"
     },
     "server_error": {
-      "message": "Connection lost",
+      "message": "$errors.server_error.message4",
       "instruction": "Please try again"
     },
     "glare_detected": {
       "message": "Glare detected",
       "instruction": "All details should be clear and readable"
     },
     "sms_failed": {
-      "message": "Something's gone wrong",
-      "instruction": "Copy the below link to your mobile instead"
+      "message": "Copy the link to your phone",
+      "instruction": "Something’s gone wrong"
     },
     "sms_overuse": {
-      "message": "Too many resend attempts",
-      "instruction": "Copy the below link to your mobile instead"
+      "message": "Copy the link to your phone",
+      "instruction": "Too many failed attempts"
```

Comment:
Same here, why are we switching the instruction and the message around?

Reply:
switched them around for the time being